fe/src/App.jsx

```diff
@@ -0,0 +1,38 @@
+import "./index.css";
```

Comment:
- ahh do not rename **App.js** file. 
- move these code to a separate file in `pages` folder and call it via router